mifos-authenticator-passcode/build.gradle.kts

```diff
@@ -0,0 +1,93 @@
+@file:OptIn(ExperimentalWasmDsl::class)
+
+import org.jetbrains.kotlin.gradle.ExperimentalWasmDsl
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+    alias(libs.plugins.kotlinMultiplatform)
+    alias(libs.plugins.androidLibrary)
+    alias(libs.plugins.compose.compiler)
+    alias(libs.plugins.composeMultiplatform)
+}
+
+kotlin {
+
+    androidTarget {
+        compilerOptions {
+            jvmTarget.set(JvmTarget.JVM_17)
+        }
+    }
+
+    listOf(
+        iosX64(),
+        iosArm64(),
+        iosSimulatorArm64()
+    ).forEach { iosTarget ->
+        iosTarget.binaries.framework {
+            baseName = "mifos-authenticator-passcode"
+            isStatic = true
+        }
+    }
+
+    js(IR) {
+        browser()
+    }
+
+    jvm("desktop") {
+        compilerOptions {
+            jvmTarget.set(JvmTarget.JVM_17)
+        }
+    }
+
+    wasmJs { browser() }
+
+    sourceSets {
+
+        androidMain.dependencies {
+            implementation(compose.preview)
+            implementation(libs.androidx.activity.ktx)
+            implementation(libs.androidx.activity.compose)
+            implementation(libs.kotlinx.coroutines.android)
+        }
+
+        commonMain.dependencies {
+            implementation(compose.components.resources)
+            implementation(compose.ui)
+            implementation(compose.runtime)
+            implementation(compose.foundation)
+            implementation(compose.material3)
+
+            implementation(project(":core:designsystem"))
+
+            // For Preview
+            implementation(compose.components.uiToolingPreview)
+
+            // Material Icons
+            implementation(libs.material3.icons)
+        }
+
+        val desktopMain by getting {
+            dependencies {
+                implementation(compose.desktop.currentOs)
+                implementation(libs.kotlinx.coroutines.swing)
+            }
+        }
+
+    }
+
+}
+
+
+android {
+    namespace = "com.mifos.passcode.mifos_authenticator_passcode"
+    compileSdk= libs.versions.android.compileSdk.get().toInt()
```

Comment:
remove the `com.mifos.passcode` just use the `mifos_authenticator_passcode`.
do the same for other module. i think Niyaj suggested you to use `snacke_case` so prefix everything with `mifos_authenticator` and then the module name